_data/gaming.yml

```diff
@@ -134,7 +134,7 @@ websites:
       doc: http://itch.io/developers/two-factor-authentication
 
     - name: League of Legends
-      url: http://leagueoflegends.com
+      url: http://oce.leagueoflegends.com/
```

Comment:
The old URL automatically redirects users to the correct server based on their location, for example, http://na,leagueoflegends.com for users in North America. This is by design, and no change needs to be made to the previous URL.